sentry-ruby/spec/sentry/client_spec.rb

```diff
@@ -209,6 +209,22 @@ def sentry_context
           expect(hash[:exception][:values][0][:value]).to eq("undefined method `[]' for nil:NilClass")
         end
       end
+
+      it "converts non-string error message" do
+        NonStringMessageError = Class.new(StandardError) do
+          def message
+            { foo: "bar" }
+          end
```

Comment:
I think this is not needed because only `detailed_message` will be called?

Reply:
ahh. good catch. let me fix this.